js/ToDoList.js

```diff
@@ -0,0 +1,58 @@
+ 
+ class ToDoList{
+
+   constructor(){
```

Comment:
render the tasks list once call the constructor to list them once reload the page